homework_1/task_02/taxes/taxes.go

```diff
@@ -0,0 +1,56 @@
+package taxes
+
+import (
+	"github.com/pkg/errors"
+)
+
+type TaxClass struct {
+	UpperThreshold  float64
+	Percentage float64
```

Comment:
Formatting is still not fixed, for example the `float64` type declarations should be aligned here.
Check this out - https://blog.golang.org/gofmt